lib/elixir/lib/calendar.ex

```diff
@@ -92,6 +92,42 @@ defmodule Date do
       {:error, :invalid_date}
     end
   end
+
+  @doc """
+  Converts a `Date` struct to an Erlang date tuple (like those produced by the `:calendar` module).
+
+  Only supports converting Dates which are in the ISO calendar,
```

Comment:
Dates should be lowercase or be `Date`s. The same for Times and NaiveDateTimes below.